.travis.yml

```diff
@@ -19,18 +19,14 @@ before_deploy:
   - docker push j0elwee/devops-trg
   - docker push j0elwee/devops-trg:$TRAVIS_COMMIT
   - docker push j0elwee/devops-trg:python-3.8-slim-buster-with-geckodriver
-  - curl https://cli-assets.heroku.com/install.sh | sh
-  - echo $HEROKU_API_KEY | docker login -u j0elwee --password-stdin registry.heroku.com
-  - docker tag j0elwee/devops-trg registry.heroku.com/devops-trg/web
-  - docker push registry.heroku.com/devops-trg/web
 deploy:
   provider: script
-  script: heroku container:release web -a devops-trg
+  script: curl -dH -X POST $AZURE_WEBHOOK
   on: 
-    branch: master
+    branch: mod-11-azure
```

Comment:
The deploy actually failed with `401 - Unauthorized: Access is denied due to invalid credentials`.

Also, don't forget to revert the branch name before merging.